### PR TITLE
chore: update Row, Column and Navigation Manager docs

### PR DESCRIPTION
**Pull request by soumyaloka**
## Description

In this PR, styles table has been updated and also improved the description of controls for Row and Column components.

## References

LUI-748

## Checklist

- [x] all commented code has been removed
- [ ] any new console issues have been resolved
- [x] code linter and formatter has been run
- [ ] test coverage meets repo requirements
- [x] PR name matches the expected semantic-commit syntax

### CODE REVIEW

File: packages/@lightningjs/ui-components/src/components/Column/Column.mdx [hunk 1]
```diff
@@ -57,7 +53,7 @@ class BasicExample extends lng.Component {
 }
 ```
 
-The `Column` component extends [FocusManager][fm-docs], so you need to make sure that `_getFocused` returns the `Column` component.
+The `Column` component extends FocusManager, so you need to make sure that `_getFocused` returns the `Column` component.
```
Comment (erautenberg) [inline packages/@lightningjs/ui-components/src/components/Column/Column.mdx]: Could you add a link to the FocusManager here? I assume that's what "fm-docs" was supposed to be.

**Review by erautenberg — APPROVED** [review]
I think we can set the `type` of `itemTransition` in Row/Column to be `Object` instead of just empty.

**Comment by soumyaloka** [comment]
> I think we can set the `type` of `itemTransition` in Row/Column to be `Object` instead of just empty.

`itemTransition`  type is `<DocsLink id="lng.Transition" />` now , I can replace that with object.

File: packages/@lightningjs/ui-components/src/components/Column/Column.mdx (hunk 1 above)
Comment (ImCoolNowRight) [reply packages/@lightningjs/ui-components/src/components/Column/Column.mdx]: Do we still want to add this link in?

**Review by arwehrman — APPROVED** [review]

**Review by ImCoolNowRight — APPROVED** [review]
Looks good! Just left a comment about the link to the FocusManager if we still want it added.

**Pull request merged**